src/main/kotlin/org/malachite/estella/queues/utils/MsgDeserializer.kt

```diff
@@ -0,0 +1,43 @@
+package org.malachite.estella.queues.utils
+
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import org.malachite.estella.commons.models.tasks.TaskResult
+import org.malachite.estella.process.domain.toTask
+import org.malachite.estella.services.TaskService
+import org.malachite.estella.services.TaskStageService
+import org.springframework.beans.factory.annotation.Autowired
+import java.sql.Timestamp
+import java.util.*
+import javax.sql.rowset.serial.SerialBlob
+import javax.sql.rowset.serial.SerialClob
+
+class MsgDeserializer(
```

Comment:
Ogólnie jeśli klasa nazywa się `MsgDeserializer` to powinna ona robić dokładnie to - deserializować message, a nie jeszcze korzystać z jakichś rep i tak dalej. Możesz to nazwać jakoś inaczej, albo, co chyba lepsze - zostawić deserializację w tej klasie do jakiejś klasy pośredniej, a odpytywanie rep zrobić w konsumentach.